frontend/src/api/recipe.ts

```diff
@@ -49,4 +57,21 @@ export const submitRecipe = async (formData: FormData) => {
     return response
 }
 
+export const submitComment = async (
+    formData: FormData,
+    recipeId: string = ''
+) => {
+    const formObject = Object.fromEntries(formData.entries())
+    const formWithData = Object.assign({}, formObject, { date: new Date() })
```

Comment:
This could be simplified using the spread syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Have a look here ;)